File: client-java/instrumentation/src/test/java/com/foo/somedifferentpackage/examples/entity/EntityY.java
package com.foo.somedifferentpackage.examples.entity;


import javax.persistence.Column;
import javax.persistence.Entity;
import javax.validation.constraints.NotNull;

@Entity(name="BAR")
public class EntityY {

    @NotNull
    public Boolean x;

    @Column(name = "foo")
    public int y;

    @NotNull
    @Column(name="hello")
    private Double z;

    private long k;
}

File: e2e-tests/spring-rest-multidb/src/main/kotlin/com/foo/rest/examples/multidb/separatedschemas/EntityY.kt
package com.foo.rest.examples.multidb.separatedschemas

import javax.persistence.Entity
import javax.persistence.Id
import javax.persistence.Table
import javax.validation.constraints.NotNull

@Entity
@Table(name = "EntityY", schema = "bar")
open class EntityY {

    @get:Id
    @get:NotNull
    open var id: String? = null
}